app/jobs/import_config_report.rb

```diff
@@ -0,0 +1,9 @@
+class ImportConfigReport < ActiveJob::Base
+  def perform(config_report, proxy)
```

Comment:
When this runs, `ConfigReport` will print `processing report for lenovolobato.lobatolan.home`, hence why I didn't include a `before_perform` method to log it.